bugzilla.py

```diff
@@ -0,0 +1,12 @@
+import utils
+
+
+def get_signatures_from_bug(bug_id):
+    url = 'https://bugzilla.mozilla.org/rest/bug'
+    response = utils.get_with_retries(url, params={'id': bug_id})
+    response.raise_for_status()
+    return response.json()['bugs'][0]['cf_crash_signature']
```

Comment:
cf_crash_signature can contain multiple signatures, we have to split the field in an array. See for example https://bugzilla.mozilla.org/show_bug.cgi?id=1333486.